src/main/java/org/apache/sysml/runtime/controlprogram/paramserv/ParamservUtils.java

```diff
@@ -77,6 +83,12 @@ public static void cleanupData(Data data) {
 		cd.clearData();
 	}
 
+	public static MatrixObject newMatrixObject() {
+		return new MatrixObject(Expression.ValueType.DOUBLE, null,
```

Comment:
Please use `OptimizerUtils.getUniqueTempFileName()` instead of null for the filename.